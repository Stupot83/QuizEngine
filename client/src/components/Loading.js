import React from "react";
import Logo from "../Logo-loading.png";
import "../sass/Loading.scss";

const Loading = () => {
    return (
        <div className="Logo_container">
            <img className="Logo_loading" src={Logo} alt="Loading..." />
        </div>
    );
};

export default Loading;
