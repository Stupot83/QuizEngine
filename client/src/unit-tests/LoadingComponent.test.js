import React from "react";
import { shallow } from "enzyme";
import Loading from "../components/Loading";

describe("Loading component", () => {
    it("should render correctly in debug mode", () => {
        const component = shallow(<Loading debug />);

        expect(component).toMatchSnapshot();
    });

    it("should render correctly", () => {
        const component = shallow(<Loading />);

        expect(component).toMatchSnapshot();
    });
});
