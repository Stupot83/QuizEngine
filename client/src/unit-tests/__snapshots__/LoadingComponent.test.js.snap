// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render correctly 1`] = `ShallowWrapper {}`;

exports[`Loading component should render correctly in debug mode 1`] = `ShallowWrapper {}`;
